src/test/kotlin/org/rust/IgnoreIntentionPreview.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust
+
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class IgnoreIntentionPreview
```

Comment:
I added this annotation for intentions which generate code and add some imports. It is quite difficult to support import insertion in the preview, and imports are not so important part of the preview, so I think for now it is ok to not show imports in the preview

Reply:
As I can see, you use this annotation in cases when preview differs from intention result. Maybe it's better introducing a test API to check custom previews instead of skipping them? Otherwise, it's quite easy to miss something

Reply:
Added `afterPreview` parameter to `RsIntentionTestBase#doAvailableTest` to check preview result. Also I have to inline `RsIntentionTestPlatformBase#checkPreviewAndLaunchAction`, so `RsIntentionTestPlatformBase` is now unused. I will remove it if review is ok

Reply:
Great!
But do we still need this annotation after the latest changes at all? it seems in such cases, you can just provide custom preview and drop `IgnoreIntentionPreview`
If we still need it, could you add some short doc comment why someone may need it?